example-thesis.tex

```diff
@@ -2,50 +2,65 @@
 \setcounter{secnumdepth}{4}
 \setcounter{tocdepth}{4}
 
-\usepackage{lipsum}
-
-\usepackage[font={small,it}]{caption}
-\usepackage{algorithm, algorithmic}
-\usepackage{subcaption}
-\usepackage{siunitx}
-\usepackage{acro}
-\usepackage[margin=1.25in]{geometry}
-
+%% Information for formatting in these files
+%% All \usepackage{} located in preamble
 \input{includes/preamble}
-\input{includes/defs}
 \input{includes/thesisdefs}
 
 \clearpage\pagenumbering{roman}  % This makes the page numbers Roman (i, ii, etc)
 
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%% EDIT THIS SECTION WITH YOUR INFO %%
+
+%% Use double spacing for depositing dissertation
+%% other options: \OnehalfSpacing, \SingleSpacing
+\DoubleSpacing
+
+%% Your info
 \title{Title of your Dissertation}
-\author{Buckingham U. Badger}
-\department{Nuclear Engineering and Engineering Physics}
+\author{Buckingham U. Badger}  % your name here
+\department{Engineering Physics}
```

Comment:
The degree program name should be in there somewhere, too.

Reply:
Hmmm I will look at the original template again and the requirements to figure out where that is supposed to go.